backend/src/main/resources/application.properties

```diff
@@ -1,20 +1,23 @@
-server.port=8080
-
 spring.profiles.active=dev
 
+# REST related config
+server.servlet.context-path = /api/v1
+
+server.port=8080
+
 spring.data.neo4j.uri=bolt://localhost:7687
-spring.data.neo4j.username=neo4j
-spring.data.neo4j.password=stonecap
+spring.data.neo4j.username=${NEO4J_USERNAME}
+spring.data.neo4j.password=${NEO4J_PASSWORD}
 
 # Alpaca Related properties
 # Client ID Auth: 198903d0d2f523a25e4dce65837bbf0d
 # Client Secret Auth: d756cb7ae52ad380d16aec004a38b00f04747cf2
 alpaca.api.version=v2
-alpaca.key.id=PKZJK3IXQ4JZAWCBDSRX
-alpaca.secret=SA5sYIYR58AukN8NKpdGkQZSCvOCoTXYgyjuaoUe
+alpaca.key.id=${ALPACA_KEY_ID}
+alpaca.secret=${ALPACA_SECRET_ID}
 alpaca.base.api.url=https://paper-api.alpaca.markets
 alpaca.base.data.url=https://data.alpaca.markets
 
 # IEXCloud related properties
-IEXCloud.publishable.token=Tpk_41cea3476f55437aa0a7ff89162ea9c4
-IEXCloud.secret.token=Tsk_072d4c09f38148a9b68b748b38fad53c
\ No newline at end of file
+IEXCloud.publishable.token=${IEXCLOUD_PUBLISHABLE_TOKEN_SANDBOX}
+IEXCloud.secret.token=${IEXCLOUD_SECRET_TOKEN_SANDBOX}
```

Comment:
If we append _SANDBOX to the env variables for iexcloud here, it would mean that we also use the sandbox keys for prod. I see 2 solutions to this:

1. We rename the environment variable by removing the sandbox suffix and use the appropriate key in the different environment
e.g. 
- local env: export IEXCLOUD_*_TOKEN = sandbox key
- ci env: export IEXCLOUD_*_TOKEN = sandbox key
- dev env: export IEXCLOUD_*_TOKEN = sandbox key
- prod env: export IEXCLOUD_*_TOKEN = prod key
 
OR

2. We have both (or just one) of prod and sandbox keys on all environment, and we decide which one we use inside the application by making use of the spring profile property (dev or prod)

Reply:
I just saw this comment, Ill update the PR